src/send-reset-pwd.js

```diff
@@ -21,17 +21,36 @@ async function sendResetPwd (options, identifyUser, field, notifierOptions = {})
   const users = await usersService.find({ query: identifyUser });
   const user1 = getUserData(users, options.skipIsVerifiedCheck ? [] : ['isVerified']);
 
+  if (
+    // Use existing token if remaining time exceeds half of resetDelay
+    options.reuseResetToken
+    && user1.resetToken
+    && user1.resetToken.includes('___')
+    && user1.resetExpires > Date.now() + options.resetDelay / 2
+  ) {
+    await notifier(options.notifier, 'sendResetPwd', user1, notifierOptions);
+    return options.sanitizeUserForClient(user1);
+  }
```

Comment:
This isn't very readable.
Could you make it more readable ? 

And why do you need to check if the `user1.resetToken.includes('___')`?

Reply:
Hey @TheSinding ! I've changed the lines and made the comment more explicit. 
`includes('___')` to make sure that we have a plain token in the user object, not the hashed one. You can see the same check [here](https://github.com/feathersjs-ecosystem/feathers-authentication-management/blob/master/src/helpers/deconstruct-id.js#L7). Some tests are failing without it. And in the real-world scenario, if someone  changes the `reuseResetToken` setting in production while some user already requested the token.

Reply:
That makes sense, thanks for the clarification mate! :+1: